README.md

```diff
@@ -278,7 +278,7 @@ Update/mutate the stock picking
 * @param (optional) result_package_name - If it corresponds to an existing package/pallet that is not in an other location, we will set it to the `result_package_id` of the operations of the picking (i.e. transfer). If the target storage format of the picking type is pallet of packages it will set `result_parent_package_id`.
 * @param (optional) package_name - Name of the package of the picking to be marked as done
 * @param (optional) products_info - An array with the products information to be marked as done, where each dictionary contains: product_barcode, qty and serial numbers if needed
-
+* @param (optional) picking_info - A dictionary of generic picking information to update the stock picking with
```

Comment:
Have you double checked if this parameter name is correct or if it has to be called "transport" and only set the transport related information?

Reply:
After discussion it was decided to leave it as picking_info and add the trailer information to it.